tensorflow/core/kernels/eigen_spatial_convolutions-inl.h

```diff
@@ -347,13 +378,131 @@ class TensorContractionInputMapper<
     if (nonStandardPatches()) {
       return packetWithPossibleZero(patchId, rowIndex, colIndex, otherIndex);
     }
-    return loadPacketStandard(patchId, rowIndex, colIndex, otherIndex);
+    typedef decltype(m_impl) TensorEvaluatorT;
+    return loadPacketStandard<Packet, TensorEvaluatorT>(patchId, rowIndex,
+                                                        colIndex, otherIndex);
+  }
+
+  // Helper function to load a 'partial' packet - this is the single column
+  // part of a packet that is split across two columns.
+  // If required, this function is called from
+  // loadPacketStandardFromTwoColumns() when the packet type supports masked
+  // load and when the partial packet load is available in the TensorEvaluator.
+  EIGEN_DEVICE_FUNC
+  EIGEN_ALWAYS_INLINE Packet loadPartialPacketStandard(
+      Index patchId, Index rowIndex, Index colIndex, Index otherIndex,
+      Index patchIdPart, const Index spanPart[], const Index patchOffsetsPart[],
+      Index colOffsetPart) const {
+    const Index inputColPart = colIndex + colOffsetPart;
+    const Index rowOffsetsPart[2] = {
+        patchOffsetsPart[0] - colOffsetPart * m_colStride,
+        patchOffsetsPart[1] - colOffsetPart * m_colStride};
+    const Index inputRowsPart[2] = {rowIndex + rowOffsetsPart[0],
+                                    rowIndex + rowOffsetsPart[1]};
+
+    if (inputRowsPart[0] >= m_inputRows || inputRowsPart[1] < 0 ||
+        inputColPart >= m_inputCols || inputColPart < 0) {
+      return internal::pset1<Packet>(Scalar(0));
+    } else if (inputRowsPart[0] >= 0 && inputRowsPart[1] < m_inputRows) {
+      const Index inputIndexPart = patchIdPart -
+                                   patchOffsetsPart[0] * patchDepth() +
```

Comment:
`patchIdPath - patchOffsetsPart[0] * patchDepth()` is extracted into `depth` variable in other code, this helps a bit with readability

Reply:
I will make this modification. Thanks.